tests/acceptance/01_vars/02_functions/string.cf

```diff
@@ -0,0 +1,41 @@
+body common control
+{
+	bundlesequence => { "init", "check" };
+}
+
+bundle agent init
+{
+	classes:
+		"ExistingClass";
+
+	vars:
+		"a_int" int => "1k";
+		"a_real" real => "6.066";
+		"a_string" string => "Really interesting string";
+
```

Comment:
`string()` should stringify a data container into a JSON serialized string, and the test should test this:

```suggestion
		"a_data_container" data => '{"foo": "bar"}';

```